### PR TITLE
feat:  deploy operator by 'make deploy'

### PR DESCRIPTION
**Pull request by 0xff-dev**

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -45,36 +45,26 @@ spec:
...
-                  fieldPath: metadata.namespace
             - name: POD_NAME
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: OPERATOR_NAMESPACE
+              value: "operator-system"
             - name: OPERATOR_NAME
-              value: operator
+              value: "fabric-operator"
```
Comment (Abirdcfly) [inline config/manager/manager.yaml:59]: use metadata.namespace

File: config/manager/manager.yaml [hunk 2]
```diff
@@ -45,36 +45,26 @@ spec:
...
             - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
             - name: POD_NAME
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: OPERATOR_NAMESPACE
+              value: "operator-system"
```
Comment (Abirdcfly) [inline config/manager/manager.yaml:57]: use metadata.namespace

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -45,36 +45,26 @@ spec:
...
             - name: CLUSTERTYPE
-              value: OPENSHIFT
-          image: todo:update
-          imagePullPolicy: Always
+              value: K8S
+            - name: OPERATOR_INGRESS_DOMAIN
+              value: 172.22.50.142.nip.io
+            - name: OPERATOR_IAM_SERVER
+              value: https://oidc-server.u4a-system.svc
+          image: hyperledgerk8s/fabric-operator:latest-amd64
```
Comment (Abirdcfly) [inline config/manager/manager.yaml:66]: controller:latest

File: config/manager/manager.yaml [hunk 4]
```diff
@@ -109,22 +99,21 @@ spec:
...
       affinity:
         nodeAffinity:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
               - matchExpressions:
-                  - key: kubernetes.io/arch
-                    operator: In
-                    values:
-                      - amd64
-                      - s390x
```
Comment (Abirdcfly) [inline config/manager/manager.yaml:121]: amd64 and arm64

File: config/manager/manager.yaml [hunk 5]
```diff
@@ -109,22 +99,21 @@ spec:
             readOnlyRootFilesystem: false
             runAsNonRoot: false
             runAsUser: 1001
+      tolerations:
```
Comment (Abirdcfly) [inline config/manager/manager.yaml:102]: delete 102 and 103?

File: config/samples/orgs/org1.yaml [hunk 6]
```diff
@@ -12,7 +12,7 @@ spec:
     license:
       accept: true
     images:
-      caImage: hyperledgerk8s/fabric-ca
+      caImage: config/rbac/role_binding.yaml/fabric-ca
```
Comment (Abirdcfly) [inline config/samples/orgs/org1.yaml:15]: ?

**Review by Abirdcfly — APPROVED** [review]
LGTM

**Comment by bjwswang** [comment]
LGTM

**Review by bjwswang — APPROVED** [review]
Since installer use deployment files, we need a mechanism to sync operator files with installer in the future.

**Pull request merged**